README.md

```diff
@@ -1,3 +1,11 @@
+[![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs)
+[![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](http://www.gnu.org/licenses/gpl-3.0)
+![Open Source Love](https://img.shields.io/badge/Open%20%E2%9D%A4%EF%B8%8FSource-blue)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/anitab-org/open-source-programs-web)
+![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)
+[![GitHub forks](https://img.shields.io/github/forks/anitab-org/open-source-programs-web?style=social)](https://github.com/anitab-org/open-source-programs-web/network/members)
+[![GitHub stars](https://img.shields.io/github/stars/anitab-org/open-source-programs-web?style=social)](https://github.com/anitab-org/open-source-programs-web/stargazers)
+
```

Comment:
These are redundant since this is present in the GitHub interface. Could you remove them please?